tests/ExchangeRateServiceTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Lidonation\CardanoPayments\Services\ExchangeRateService;
+
+it('can verify currency support', function () {
+    $service = new ExchangeRateService();
+    $supported = $service->verifyCurrencySupport(['ADA', 'USD', 'EUR', 'HOSKY']);
+    $this->assertTrue($supported);
+});
+
+it('can get exchange rates', function () {
```

Comment:
@michaelteddy009 please make this a more meaningful test. Maybe this scenario:
1. Mock the ExchangeRateService class
2. Override the methods that return exchange rates to return predictable rates
3. Update your test to assert against an exact float value


ie:
```
# assume that the ada/usd pairs from the three services are .53 USD to 1 Ada, .49:1, .51:1. (Mock the ExchangeRateService class so you can control the pairs, and so the  test can run interdependently without requiring internet connection)  
# then assert that given 50USD the function will return 98.03
```